### ISSUE TITLE
OOM Memory issue

### ISSUE THREAD

**Issue by HSinger04** [HSinger04/VOGUE-Reimplementation#13]
Try ResNet50 instead of VGG. Reason: https://www.researchgate.net/figure/Comparison-of-popular-CNN-architectures-The-vertical-axis-shows-top-1-accuracy-on_fig2_320084139 and https://arxiv.org/pdf/2006.12655.pdf

**Comment by HSinger04** [comment]
From https://www.tensorflow.org/api_docs/python/tf/keras/applications:

EfficientNetB0 < ResNet50 < InceptionResNetv2
